project.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<p>Highly useful? How useful is that?</p><p> </p><p>1 level of quality away... interesting metric to track. Explain why this is very important. You claim classification works well from observing it's accuracy (perfectly predicting the correct label), but if an example has a label of 5 is it better to predict of 4 or 1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-42FraMeWork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>